eva_sub_cli/validators/validator.py

```diff
@@ -183,6 +176,43 @@ def _collect_validation_workflow_results(self):
         self._load_fasta_check_results()
         self._collect_metadata_results()
 
+    def _asses_validation_results(self):
+        """
+            Assess if the validation results are meeting expectations and marks them as "PASS: true" or "PASS: false"
+            It assumes all validation have been parsed already.
+        """
+        # vcf_check result
+        vcf_check_result = all((vcf_check.get('critical_count', 1) == 0
+                                for vcf_name, vcf_check in self.results.get('vcf_check', {}).items()))
+        self.results['vcf_check']['PASS'] = vcf_check_result
+
+        # assembly_check result
+        asm_nb_mismatch_result = all((asm_check.get('nb_mismatch', 1) == 0
+                                      for vcf_name, asm_check in self.results.get('assembly_check', {}).items()))
+        asm_nb_error_result = all((asm_check.get('nb_error', 1) == 0
+                                   for vcf_name, asm_check in self.results.get('assembly_check', {}).items()))
+        self.results['assembly_check']['PASS'] = asm_nb_mismatch_result and asm_nb_error_result
+
+        # fasta_check result
+        fasta_check_result = all((fa_file_check.get('all_insdc', False) is True
+                                  for fa_file, fa_file_check in self.results.get('fasta_check', {}).items()))
+        self.results['fasta_check']['PASS'] = fasta_check_result
+
+        # sample check result
+        self.results['sample_check']['PASS'] = self.results.get('sample_check', {}).get('overall_differences',
+                                                                                        True) is False
+
+        # metadata check result
+        metadata_xlsx_result = len(self.results.get('metadata_check', {}).get('spreadsheet_errors', [])) == 0
+        metadata_json_result = len(self.results.get('metadata_check', {}).get('json_errors', [])) == 0
+        self.results['metadata_check']['PASS'] = metadata_xlsx_result and metadata_json_result
+
+    def _save_validation_results(self):
+        with open(self.validation_result_file, 'w') as val_res_file:
+            yaml.safe_dump(self.results, val_res_file)
+
+        self.info(f"saved validation result in {self.validation_result_file}")
```

Comment:
Just to minimise logging for users:
```suggestion
        self.debug(f"Saved validation result in {self.validation_result_file}")
```